balancer/roundrobin/roundrobin.go

```diff
@@ -33,6 +33,8 @@ import (
 // Name is the name of round_robin balancer.
 const Name = "round_robin"
 
+var logger = grpclog.Component("BALANCER_ROUNDROBIN")
```

Comment:
Do we want the component name to be all caps?
How about just `balancer_roundrobin`?

Reply:
As discussed offline: let's try `RoundRobin` or `roundrobin` to avoid long names.